ci/test_python.sh

```diff
@@ -136,23 +129,23 @@ else
 fi
 
 if [[ "${RUNNER_ARCH}" != "ARM64" ]]; then
+  rapids-logger "Generate Python testing dependencies"
+  rapids-dependency-file-generator \
+    --output conda \
+    --file-key test_pylibwholegraph \
+    --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}"  \
+    --prepend-channel "${CPP_CHANNEL}" \
+    --prepend-channel "${PYTHON_CHANNEL}" \
+    --prepend-channel pytorch \
```

Comment:
Not a blocking comment for this PR, but @tingyu66 @alexbarghi-nv are we planning to drop use of the `pytorch` channel in the 25.04 release?

Linking this related conversation: https://github.com/rapidsai/cugraph-gnn/pull/99#discussion_r1892492252

Reply:
Thank you! I can get that started in a follow-up PR if you'd like.